autorest/adal/token_test.go

```diff
@@ -722,6 +723,22 @@ func TestGetVMEndpoint(t *testing.T) {
 	}
 }
 
+func TestGetAppServiceEndpoint(t *testing.T) {
```

Comment:
Can you also add a test (or update this one, whatever makes sense) to cover `GetMSIEndpoint()`.

Reply:
I've added a tests for both VM and App Service results in the main `GetMSIEndpoint()` function. I also cleaned up the environment variable handling in tests.